installer/base.py

```diff
@@ -29,6 +30,18 @@ class InstallerError(Exception):
     pass
 
 
+# Type annotation not accepting Network maybe because the Network class is declared later
```

Comment:
The Network class is only defined later on the file, that is why the typing system could not find it at this point.

You can do either of the folllowing:
    1. move these functions to the bottom of the file (after the definition of the relevant types)
    2. move these functions to a separate file/module 
    3. keep the functions where they are, but use quotes on the type annotation. This way the python parser will know that it is a type that will be resolved at some later point.

I'd favor (2), as it seems that these functions are more close to infura-specific logic and defined to help on the presentation part (not really part of the core logic)